tools/test-api

```diff
@@ -55,6 +60,15 @@ with test_server_running(
     email = "iago@zulip.com"  # Iago is an admin
     realm = get_realm("zulip")
     user = get_user(email, realm)
+
+    # Iago needs permission to manage all user groups.
+    admins_group = NamedUserGroup.objects.get(
+        name=SystemGroups.ADMINISTRATORS, realm=realm, is_system_group=True
+    )
+    do_change_realm_permission_group_setting(
+        realm, "can_manage_all_groups", admins_group, acting_user=None
+    )
```

Comment:
I'm not sure if there's a better way to do this directly in zulip.yaml, any inputs appreciated.